cmd/root.go

```diff
@@ -325,6 +328,28 @@ func initConfig() {
 			)
 		}
 	}
+
+	podSyncPoolSize = getPodSyncPoolSize(logger)
+}
+
+func getPodSyncPoolSize(logger *logrus.Entry) int {
+	poolSizeEnv := os.Getenv("POD_SYNC_POOL_SIZE")
```

Comment:
Why not a flag instead of an environment var?

Reply:
"--pod-sync-workers"

Reply:
I'm also struggling about using a flag vs. env var. Sounds flag makes more sense as it's a configuration to the VK itself. Will switch to it.